fk/keycreate.go

```diff
@@ -248,11 +250,18 @@ func verifyEmailMatchesKeyInAPI(
 
 	retrievedKey, err := pgpkey.LoadFromArmoredPublicKey(armoredKey)
 	if err != nil {
-		return false, fmt.Errorf("Failed to load armored key: %v", err)
+		return false, fmt.Errorf("failed to load armored key: %v", err)
 	}
 	if retrievedKey.Fingerprint() == fingerprint {
 		return true, nil
 	}
+	retrievedEmail, err := retrievedKey.Email()
+	if err != nil {
+		return false, fmt.Errorf("couldn't get email for retrieved key: %s", err)
+	}
+	if retrievedEmail == email {
+		return false, fmt.Errorf("a key for %s is already verified\n     Please email security@fluidkeys.com and we can manually remove the old key\n", retrievedEmail)
+	}
 	return false, fmt.Errorf("Retrieved key's fingerprint doesn't match")
```

Comment:
*this* is the line you probably meant to replace... right?

I'd suggest flipping the logic first so the failure case comes first, and the success case comes last (I did this on a WIP branch: 69b96db2b47337e9e27d075bb35366b9664f807d)